server/handlers.go

```diff
@@ -134,3 +134,81 @@ func (server HTTPServer) getRuleIDs(writer http.ResponseWriter, request *http.Re
 		return
 	}
 }
+
+// overviewEndpoint returns a map with an overview of number of clusters hit by rules
+func (server HTTPServer) overviewEndpoint(writer http.ResponseWriter, request *http.Request) {
+	authToken, err := server.GetAuthToken(request)
+	if err != nil {
+		handleServerError(writer, err)
+		return
+	}
+
+	userID := authToken.AccountNumber
+	orgID := authToken.Internal.OrgID
+
+	clustersHits := 0
+	var hitsByTotalRisk map[int]int = make(map[int]int)
```

Comment:
you actually don't need `var` and `map[int]int` here :)

Reply:
Ack